clang/include/clang/AST/Type.h

```diff
@@ -691,6 +691,10 @@ class QualType {
   unsigned getLocalFastQualifiers() const { return Value.getInt(); }
   void setLocalFastQualifiers(unsigned Quals) { Value.setInt(Quals); }
 
+  void QualifyAsCheckedPtr() const;
```

Comment:
**Explanation:**
These new functions are used to communicate Type Qualifier information from DeclSpec to the QualType. This is done in the SemaType.cpp file in this pull request.